src/index.js

```diff
@@ -80,7 +80,7 @@ require('./extras/primitives/');
 
 console.log('A-Frame Version: 1.5.0 (Date 2024-05-03, Commit #9fe641ce)');
 console.log('THREE Version (https://github.com/supermedium/three.js):',
-            pkg.dependencies['super-three']);
+            pkg.dependencies['three']);
```

Comment:
This would now log the npm alias `npm:super-three@0.164.0` which isn't really informative to the end user. We could consider `THREE.REVISION` instead, although that wouldn't include patch version AFAIK, so for example `0.164.1` would simply become `164`

Reply:
I agree, THREE.REVISION is probably enough here. I'll make the change.

Reply:
Or I can just remove the "npm:super-three@" part with `pkg.dependencies['three'].replace('npm:super-three@', '')`
so that stays the same.

Reply:
Actually it's better THREE.REVISION. I have potentially a more recent version of super-three with patches in my project, and returning the version specified in aframe dependency is actually confusing.